source/exact.d.ts

```diff
@@ -0,0 +1,52 @@
+import {Primitive} from './primitive';
+import {KeysOfUnion} from './internal';
+
+/**
+Create a type from type A and B and changes keys exclusive to type B to `never`.
+
+This is useful for function type-guarding to reject arguments with excess
+properties. Due to the nature of TypeScript, it does not complain if excess properties are
+provided unless the provided value is an object literal.
+
+@example
+```
+type OnlyAcceptName = {name: string};
+
+function onlyAcceptName(args: OnlyAcceptName) {}
+
+// TypeScript complains this because it's an object literal.
```

Comment:
Typo

Reply:
> > TypeScript complains this because it's an object literal.
> Typo

I am not sure which word has typo but I rewrite this sentence and hopefully this makes more sense.

I have updated the other comments as suggested.

Reply:
I would recommend using a grammar checker like Grammarly. It would have caught this.